tests/sqlite3/test-sqlite3.cpp

```diff
@@ -413,6 +413,233 @@ TEST_CASE("SQLite long long", "[sqlite][longlong]")
     CHECK(v2 == v1);
 }
 
+// Test the DDL and metadata functionality
+TEST_CASE("SQLite DDL with metadata", "[sqlite][ddl]")
+{
+    if (sqlite3_libversion_number() < 3036000) {
+        if (sqlite3_libversion_number() < 3014000) {
+            WARN("SQLite requires at least version 3.14.0 for column description, detected " SQLITE_VERSION);
```

Comment:
This should be something like
```suggestion
            WARN("SQLite requires at least version 3.14.0 for column description, detected " << sqlite3_version());
```